roachtest: pgjdbc failed

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2488234&tab=buildLog) on [release-20.2@0e2346a8a8a8678ad6011e0da756fa4564147994](https://github.com/cockroachdb/cockroach/commits/0e2346a8a8a8678ad6011e0da756fa4564147994):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgjdbc/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,pgjdbc.go:175,pgjdbc.go:188,test_runner.go:755: 
		Tests run on Cockroach v20.2.0-279-g0e2346a8a8
		Tests run against pgjdbc REL42.2.9
		5764 Total Tests Run
		4855 tests passed
		909 tests failed
		50 tests skipped
		122 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testTruncatedUTF8Decode - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testBadUTF8Decode - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testUTF8Decode - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2488234&tab=artifacts#/pgjdbc)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2506276&tab=buildLog) on [release-20.2@0de15384d560d1b4f09315d88204b6b2d7dfb32c](https://github.com/cockroachdb/cockroach/commits/0de15384d560d1b4f09315d88204b6b2d7dfb32c):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgjdbc/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,pgjdbc.go:175,pgjdbc.go:188,test_runner.go:755: 
		Tests run on Cockroach v20.2.0-322-g0de15384d5
		Tests run against pgjdbc REL42.2.9
		5764 Total Tests Run
		4855 tests passed
		909 tests failed
		50 tests skipped
		122 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testTruncatedUTF8Decode - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testBadUTF8Decode - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testUTF8Decode - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2506276&tab=artifacts#/pgjdbc)
Related:
- #57603 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2515143&tab=buildLog) on [release-20.2@82e9f59f73098e3f6d5a0684526216df456df690](https://github.com/cockroachdb/cockroach/commits/82e9f59f73098e3f6d5a0684526216df456df690):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgjdbc/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,pgjdbc.go:175,pgjdbc.go:188,test_runner.go:755: 
		Tests run on Cockroach v20.2.0-338-g82e9f59f73
		Tests run against pgjdbc REL42.2.9
		5764 Total Tests Run
		4855 tests passed
		909 tests failed
		50 tests skipped
		122 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testUTF8Decode - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testTruncatedUTF8Decode - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testBadUTF8Decode - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2515143&tab=artifacts#/pgjdbc)
Related:
- #57603 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2519815&tab=buildLog) on [release-20.2@2308ed0493a3f0b77f1e0412a6e18a4ebe0fb7d2](https://github.com/cockroachdb/cockroach/commits/2308ed0493a3f0b77f1e0412a6e18a4ebe0fb7d2):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgjdbc/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,pgjdbc.go:175,pgjdbc.go:188,test_runner.go:755: 
		Tests run on Cockroach v20.2.3-29-g2308ed0493
		Tests run against pgjdbc REL42.2.9
		5764 Total Tests Run
		4855 tests passed
		909 tests failed
		50 tests skipped
		122 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testBadUTF8Decode - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testUTF8Decode - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testTruncatedUTF8Decode - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2519815&tab=artifacts#/pgjdbc)
Related:
- #57603 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2544605&tab=buildLog) on [release-20.2@67442afea27cdea65e6abfa195496e0760b8d340](https://github.com/cockroachdb/cockroach/commits/67442afea27cdea65e6abfa195496e0760b8d340):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgjdbc/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,pgjdbc.go:175,pgjdbc.go:188,test_runner.go:755: 
		Tests run on Cockroach v20.2.3-63-g67442afea2
		Tests run against pgjdbc REL42.2.9
		5764 Total Tests Run
		4857 tests passed
		907 tests failed
		50 tests skipped
		122 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.postgresql.test.jdbc2.CursorFetchTest.testMultiRowResultPositioning[binary = REGULAR] - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2544605&tab=artifacts#/pgjdbc)
Related:
- #57603 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2546151&tab=buildLog) on [release-20.2@5f5f68c4248aa2f8bd873db291b8d27743880e9c](https://github.com/cockroachdb/cockroach/commits/5f5f68c4248aa2f8bd873db291b8d27743880e9c):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgjdbc/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,pgjdbc.go:175,pgjdbc.go:188,test_runner.go:755: 
		Tests run on Cockroach v20.2.3-65-g5f5f68c424
		Tests run against pgjdbc REL42.2.9
		5764 Total Tests Run
		4855 tests passed
		909 tests failed
		50 tests skipped
		122 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testTruncatedUTF8Decode - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testUTF8Decode - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testBadUTF8Decode - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2546151&tab=artifacts#/pgjdbc)
Related:
- #57603 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2547714&tab=buildLog) on [release-20.2@5f5f68c4248aa2f8bd873db291b8d27743880e9c](https://github.com/cockroachdb/cockroach/commits/5f5f68c4248aa2f8bd873db291b8d27743880e9c):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgjdbc/run_1
	orm_helpers.go:228,orm_helpers.go:154,java_helpers.go:216,pgjdbc.go:175,pgjdbc.go:188,test_runner.go:755: 
		Tests run on Cockroach v20.2.3-65-g5f5f68c424
		Tests run against pgjdbc REL42.2.9
		5764 Total Tests Run
		4855 tests passed
		909 tests failed
		50 tests skipped
		122 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testBadUTF8Decode - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testUTF8Decode - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.DatabaseEncodingTest.testTruncatedUTF8Decode - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2547714&tab=artifacts#/pgjdbc)
Related:
- #57603 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/pgjdbc_blocklist.go
+++ b/pkg/cmd/roachtest/pgjdbc_blocklist.go
@@ -4426,4 +4426,7 @@ var pgjdbcIgnoreList19_2 = blocklist{
 	"org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testBatchWithReWrittenSpaceAfterValues[3: autoCommit=NO, binary=FORCE]":              "54477",
 	"org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testReWriteDisabledForPlainBatch[2: autoCommit=NO, binary=REGULAR]":                  "54477",
 	"org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testReWriteDisabledForPlainBatch[3: autoCommit=NO, binary=FORCE]":                    "54477",
+	"org.postgresql.test.jdbc2.DatabaseEncodingTest.testBadUTF8Decode":                                                                              "54477",
+	"org.postgresql.test.jdbc2.DatabaseEncodingTest.testTruncatedUTF8Decode":                                                                        "54477",
+	"org.postgresql.test.jdbc2.DatabaseEncodingTest.testUTF8Decode":                                                                                 "54477",
 }

